README.md

```
@@ -6,156 +6,335 @@ TokenLens
+  async delete(key: string) {
+    await redis.del(key);
+  }
+};
+
+const tokenlens = new Tokenlens({ 
+  cache: redisCache,
+  ttlMs: 60 * 60 * 1000, // 1 hour
+});
 ```
 
+## Type Exports
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)